.github/workflows/main.yml

```diff
@@ -1,22 +1,22 @@
 name: Binary Release
 
-on: 
-  release:
-    types: [created]
+on:
+  push:
+    tags:
+      - 'v*'
 
 jobs:
   releases-matrix:
     name: Release Go Binary
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        # build and publish in parallel: linux/amd64, windows/amd64, darwin/amd64
-        goos: [linux, windows, darwin]
-        goarch: [386, amd64]
     steps:
     - uses: actions/checkout@v2
-    - uses: wangyoucao577/go-release-action@master
+    - uses: actions/setup-go@v2
       with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
-        goos: ${{ matrix.goos }}
-        goarch: ${{ matrix.goarch }}
+        go-version: 1.15
```

Comment:
I thought this required go 1.16?

Reply:
Hi - I skipped using 1.16 here because go.mod still uses 1.15 and I see you have a PR for changing this.

Reply:
Sounds good